tensorboard/webapp/feature_flag/actions/feature_flag_actions.ts

```diff
@@ -43,3 +43,16 @@ export const featureFlagOverrideChanged = createAction(
     flags: Partial<FeatureFlags>;
   }>()
 );
+
+export const resetFeatureFlagOverrides = createAction(
```

Comment:
I have been told the name of the action should state what action has been taken as opposed to the effect that will take place. This is a subtle difference. Maybe change this to FlagSetToDefault or something along those lines.

Reply:
What do you think of `featureFlagOverridesReset`?